census/src/main/scala/hmda/census/model/CbsaLookup.scala

```diff
@@ -34,8 +34,10 @@ object CbsaLookup extends CbsaResourceUtils {
     }
   }.toSeq
 
-  val codeMap: Map[String, String] = {
-    values.map(x => (x.cbsa, x.cbsaTitle)).toMap
+  def nameFor(cbsaId: String): String = {
+    values.find(_.cbsa == cbsaId)
```

Comment:
I have a strong opinion about "performance". In the absence of an objective measurement that proves a performance issue, there isn't a performance issue. So I'm ok with this change based on what I see in the code